ui/raidboss/data/timelines/ridorana_lighthouse.txt

```diff
@@ -0,0 +1,264 @@
+# The Ridorana Lighthouse
+
+hideall "--reset--"
+hideall "--sync--"
+
+0 "--reset--" sync /:Removing combatant Famfrit, The Darkening Cloud\.  Max HP: \d{8}\./ window 10000 jump 0
```

Comment:
\d{8} is a much nicer way to put this in case they ever change HPs.  Good thought!